web_validation_dialog/static/src/js/web_validation_dialog.js

```diff
@@ -0,0 +1,132 @@
+odoo.define("web_validation_dialog.ValidationDialog", function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var FormWidget = require('web.form_widgets');
+    var Dialog = require('web.Dialog');
+    var Model = require('web.DataModel');
+    var framework = require('web.framework');
+    var pyeval = require('web.pyeval');
+
+    var QWeb = core.qweb;
+    var _t = core._t;
+
+    FormWidget.WidgetButton.include({
+
+        /*
+         * @param field_manager: get field Value
+         * @param node: get node value
+         */
+
+        init: function (field_manager, node) {
+            this._super(field_manager, node);
+            this.options = pyeval.py_eval(this.node.attrs.options || '{}');
+            this.is_dialog_security =
+                this.options.security ? this.options.security : false;
+        },
+
+        /*
+         * @return this function return on_confirmed method
+         */
+
+        exec_action: function () {
+            var self = this;
+            if (self.node.attrs.confirm && self.is_dialog_security) {
+                Dialog.confirm(self, self.node.attrs.confirm, {
```

Comment:
in this case you don't return anything, is this correct given that you expect a callback here `this.view.recursive_save().then(self.exec_action())`?

Reply:
@simahawk here If Password is confirmed then Open change Password view, No need to return anything.

![peek 2018-11-16 12-09](https://user-images.githubusercontent.com/17664321/48602295-ad80cb80-e998-11e8-9640-5c1f82a51bcd.gif)